### PR TITLE
Fix nwb property save/load

### PR DESCRIPTION
**Pull request by alejoe91**
https://github.com/SpikeInterface/spikeextractors/issues/390 

pynwb 1.2.1

@luiztauffer can you double check this?

### CODE REVIEW

**Comment by luiztauffer** [comment]
sure!

**Comment by luiztauffer** [comment]
I think it's working well now!
`pynwb=1.2.1`
with
```python
import spikeextractors as se
import pynwb

# Write
rec, sort = se.example_datasets.toy_example()
se.NwbRecordingExtractor.write_recording(rec, 'si_test.nwb')

# Read
with pynwb.NWBHDF5IO('si_test.nwb', 'r') as io:
    nwbfile = io.read()
    print(nwbfile.electrodes['rel_x'][:])
    print(nwbfile.electrodes['rel_y'][:]) 
    print(rec.get_channel_locations())
    
    print('')
    
    print(nwbfile.electrodes['group_name'][:])
    print(rec.get_channel_groups())
```

I get
```python
[1. 2. 3. 4.]
[0. 0. 0. 0.]
[[1. 0.]
 [2. 0.]
 [3. 0.]
 [4. 0.]]

['0' '0' '0' '0']
[0 0 0 0]
```

**Comment by luiztauffer** [comment]
also the other way (reading from nwb) works fine

```python
rec2 = se.NwbRecordingExtractor('si_test.nwb')
print(rec2.get_channel_groups())
print(rec2.get_channel_locations())
```

gives 
```python
[0 0 0 0]
[[1. 0.]
 [2. 0.]
 [3. 0.]
 [4. 0.]]
```

**Comment by alejoe91** [comment]
Should I add a version check on pynwb?

**Comment by luiztauffer** [comment]
> Should I add a version check on pynwb?

In the code we have it already
https://github.com/SpikeInterface/spikeextractors/blob/eadd3e20d59a11ca32b98808a8724df0c6b5aa3e/spikeextractors/extractors/nwbextractors/nwbextractors.py#L347-L351

maybe for the requirements `pynwb<1.3.0` ?

File: spikeextractors/extractors/nwbextractors/nwbextractors.py [hunk 1]
```diff
@@ -515,6 +515,9 @@ def write_recording(recording, save_path, metadata=None):
         '''
         check_nwb_install()
 
+        if distutils.version.LooseVersion(pynwb.__version__) >= '1.3.0':
```
Comment (alejoe91) [inline spikeextractors/extractors/nwbextractors/nwbextractors.py:518]: @luiztauffer I added this check for the moment to avoid the error

**Pull request merged**